src/main/java/Duke.java

```diff
@@ -1,10 +1,221 @@
+import java.util.Scanner;
+
 public class Duke {
+    private static final int ERR_OUT_OF_BOUNDS = -2;
+    private static final int ERR_WRONG_FORMAT = -1;
+    private static final int INVALID_DONE = -1;
+    private static final int EXIT = 0;
+    private static final int LIST = 1;
+    private static final int MARK = 2;
+    private static final int ADD = 3;
+    private static final int ADD_TODO = 4;
+    private static final int ADD_DEADLINE = 5;
+    private static final int ADD_EVENT = 6;
+
+    public static Task[] Tasks = new Task[100];
+
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        printGreeting();
+        loopCommands();
+        sayGoodbye();
+    }
+
+    public static int getCommand(String line) {
+        String[] subStrings = line.split(" ");
+        if (subStrings[0].equalsIgnoreCase("bye")) {
+            return EXIT;
+        }
+        if (subStrings[0].equalsIgnoreCase("done")) {
+            return MARK;
+        }
+        if (subStrings[0].equalsIgnoreCase("list")) {
+            return LIST;
+        }
+        return ADD;
+    }
+    public static int getItemType(String line) {
+        if(line.equalsIgnoreCase("todo")) {
```

Comment:
Inconsistent spacing after if statement, before the conditional. Also seen with some for loops.